src/main/kotlin/racingcar/Car.kt

```diff
@@ -0,0 +1,24 @@
+package racingcar
+
+import kotlin.random.Random
+
+data class Car(var distance: Int = 0) {
+
+    fun moveRandom(number: Int = Random.nextInt(Companion.MAX_RANDOM_VALUE)) {
+        if (canMove(number)) {
+            distance += 1
+        }
+    }
+
+    private fun canMove(number: Int): Boolean {
+        if (number >= 4) {
+            return true
+        }
+
+        return false
+    }
+
+    companion object {
+        const val MAX_RANDOM_VALUE = 10
```

Comment:
```suggestion
        private const val MAX_RANDOM_VALUE = 10
```
내부에서 값을 사용하는 경우, 외부에서 접근하지 못하도록 private 접근자를 붙여보면 어떨까요? (이하 관련 내용 동일)